tests/unittests/test_custom_domain.py

```diff
@@ -27,3 +27,19 @@ def test_config_with_domain(self, mock_verify_access):
 
         # Verify domain in client is from config
         self.assertEqual(test_client.base_url, mock_config["base_url"])
+
+    def test_prepare_url(self, mock_verify_access):
+        """
+        Test if the correct params are added to url
+        """
+        mock_config = {'repository': 'singer-io/test-repo', "base_url": "http://CUSTOM-git.com", "access_token": "", "max_per_page": 35}
```

Comment:
It's not mock. It's just a test config.
```suggestion
        config = {'repository': 'singer-io/test-repo', "base_url": "http://CUSTOM-git.com", "access_token": "", "max_per_page": 35}
```